frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/hoc/MobileView/MobileSearchResults.js

```diff
@@ -0,0 +1,101 @@
+import React, { useMemo } from 'react'
+import { useTranslation } from 'react-i18next';
+import { Details } from "../../molecules/DetailsCard";
+import { Link } from "react-router-dom";
+import NoResultsFound from "../../atoms/NoResultsFound";
+import { Loader } from "../../atoms/Loader";
+import _ from "lodash";
+
+const MobileSearchResults = ({ config, data, isLoading, isFetching,fullConfig }) => {
+    const {apiDetails} = fullConfig
+    const { t } = useTranslation();
+    const resultsKey = config.resultsJsonPath
+    let searchResult = data?.[resultsKey]?.length>0 ? data?.[resultsKey] : []
+    searchResult = searchResult.reverse()
+    //reversing reason -> for some reason if we enable sorting on columns results from the api are reversed and shown, for now -> reversing the results(max size 50 so not a performance issue)
+    
+    // if (fullConfig?.postProcessResult){
+    //     var { isPostProcessFetching,
+    //         isPostProcessLoading,
+    //         combinedResponse }  =  Digit?.Customizations?.[apiDetails?.masterName]?.[apiDetails?.moduleName]?.postProcess(searchResult) 
+
+    //     if(combinedResponse?.length > 0){
+    //         searchResult = combinedResponse
+    //     } 
+    // }
+    
+    const propsMobileInboxCards = useMemo(() => {
+      if (isLoading) {
+        return [];
+      }
+      let cardData =  searchResult.map((row) => {
+          let mapping = {};
+          let cols = config?.columns;
+          for(let columnIndex = 0; columnIndex<cols?.length; columnIndex++) {
+              console.log("KEY FROM CONFIG : ", cols[columnIndex]?.label);
+              console.log("KEY FROM FORM_DATA : ", cols[columnIndex]?.jsonPath);
+              mapping[t(cols[columnIndex]?.label)] = t(row?.[cols[columnIndex]?.jsonPath]) || t("ES_COMMON_NA")
```

Comment:
Use ladash get function here to get value based on json path , also no need to use t() in case of value
Verify for all other screens muster, estimate, accordingly will need to make changes in all configs for mobile code